src/Templates/Microsoft.NetCore.VB.ProjectTemplates/ClassLibrary.vstemplate

```diff
@@ -2,7 +2,7 @@
 <VSTemplate Version="3.0.0" Type="ProjectGroup" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" xmlns:sdk="http://schemas.microsoft.com/developer/vstemplate-sdkextension/2010">
   <TemplateData>
     <Name Package="{860A27C0-B665-47F3-BC12-637E16A1050A}" ID="9"/>
-    <Description Package="{860A27C0-B665-47F3-BC12-637E16A1050A}" ID="6"/>
+    <Description Package="{860A27C0-B665-47F3-BC12-637E16A1050A}" ID="10"/>
```

Comment:
Good catch this is the [C# project system package](http://index/?leftProject=RoslynProjectSystem(Repo)&file=src%5CMicrosoft.VisualStudio.ProjectSystem.CSharp.VS%5CPackaging%5CCSharpProjectSystemPackage.cs&line=40).  It should be `{D15F5C78-D04F-45FD-AEA2-D7982D8FA429}` (The [VB project system package](http://index/?rightProject=RoslynProjectSystem(Repo)&file=src%5CMicrosoft.VisualStudio.ProjectSystem.VisualBasic.VS%5CPackaging%5CVisualBasicProjectSystemPackage.cs&line=14)).  Looks like this was wrong even when the templates lived in the SDK repo